tests/benchmarks/android_geekbench.py

```diff
@@ -161,21 +158,18 @@ def setupGovernor(self):
         'COMPUTE'
 ]
 
-# Reboot device only the first time
-do_reboot = True
 tests_remaining = len(governors) * len(tests)
 tests_completed = 0
 for governor in governors:
     for test in tests:
         tests_remaining -= 1
         delay_after_s = 30 if tests_remaining else 0
         try:
-            GeekbenchTest(governor, test, do_reboot, delay_after_s)
+            GeekbenchTest(governor, test, delay_after_s)
```

Comment:
Mmm... with the new "iterations_pause" parameter are not we adding a pause before every iterations, even if it's just one?

If that's the case I think, with the new API you added, here we can get ride of `delay_after_s` as well... and the redefinition of the `benchmarkFinalize()` which are usually used just to add this sleep after a test has completed.

This would make this patch even more nicely red colored... and @JaviMerino (do you remember Javi?) will be one more time your best friend! ;-)